packages/cloudflare/package.json

```diff
@@ -69,6 +70,7 @@
     "@dotenvx/dotenvx": "catalog:"
   },
   "peerDependencies": {
+    "rimraf": "^6.0.1",
```

Comment:
Probably worth moving this to the workspace file for consistency (the two eslint ones above might need moving too)